client/src/locales/en.json

```diff
@@ -67,45 +66,57 @@
-  "authorization.input.email.invalid": "Please enter a valid email address.",
-  "authorization.input.password.label": "Password",
-  "authorization.input.password.confirm": "Confirm password",
-  "authorization.input.password.invalid": "Please use min 4 and max 32 signs and don't use space characters.",
-  "authorization.input.password.not-match": "Passwords do not match.",
-  "authorization.actions.sign-up.confirmation-link-sent": "In order to complete sign up process please click the link that we have just sent to {data}.",
-  "authorization.actions.sign-up.user-already-exist": "User of email {data} already exist.",
-  "authorization.sign-up.result-success": "Account created. Please go to the {link} and sign in to app.",
-  "authorization.sign-up.result-failed": "Activation link has expired or account is already activated.",
-  "authorization.sign-up.result-link": "sign in/sign up page",
-  "authorization.actions.sign-in.invalid-credentials": "Wrong email or password. Try again or click {link} to reset password.",
-  "authorization.reset-password.heading": "Reset your password",
-  "authorization.reset-password.email-label": "Enter email",
-  "authorization.reset-password.button-content": "Reset password",
-  "authorization.actions.reset": "An email with instructions to reset password has been sent to {data} if it is associated with an EOC account. Your existing password has not been changed.",
-  "authorization.password-recovery-success": "Password successfully updated. Please go to the {link} and sign in to app.",
-  "authorization.pass-recovery-form.heading": "Set new password",
-  "authorization.actions.recovery-password-default-error": "Account does not exist or is not activated.",
+  "user.name": "Name",
+  "user.email": "Email",
+  "user.password": "Password",
+  "user.photo": "Photo",
+  "user.profile": "Profile",
+  "user.profile.account": "Account info",
+  "user.profile.personal-info": "Personal info",
+  "user.profile.contact-info": "Contact info",
+  "user.profile.change-password": "Change password",
+  "user.profile.account-activation": "Account activated on",
+  "user.profile.account-activation-date": "{date} at {time}",
+  "user.auth-box.description": "The best app to track your office inventory.",
+  "user.auth-box.sign-in": "Sign in to app:",
+  "user.auth-box.sign-in-google": "Sign in with Google",
+  "user.auth-box.cta": "Try it out:",
+  "user.auth-box.demo-button": "Live demo",
+  "user.auth-box.sign-up": "Sign up to app:",
+  "user.auth-box.create-account": "Create account",
+  "user.auth.sign-in": "Sign in",
+  "user.auth.sign-up": "Sign up",
+  "user.auth.change": "Change",
+  "user.auth.sign-up.button.resend-link": "Resend link",
+  "user.auth.forgot-password.question": "If you forgot the password please click {link} to reset it.",
+  "user.auth.forgot-password": "Forgot password",
+  "user.auth.input.current-password": "Current password",
+  "user.auth.input.username.invalid": "Please use min 1 and max 32 chars.",
```

Comment:
I think this should be parameterized with `{min}` and `{max}` values.

Reply:
Message id is passed as a externaleErrorId prop to AuthInput component, which is used in many places. I would have to add something like errorParams and pass an object with those params  to use them within AuthInpu component. 
But not every components will have them, so I'll have to make some ugly checking if errorParams object exist etc, then....create some `Lets` to store those values if they exist and pass them down to `formatMessage()` function 

![SignUpForm_jsx_—_eoc](https://user-images.githubusercontent.com/27632432/62514440-7fd9fc00-b81f-11e9-8c67-b14aebeb30d6.png)

![AuthInput_jsx_—_eoc](https://user-images.githubusercontent.com/27632432/62514439-7f416580-b81f-11e9-87a9-d1aa61227fd2.png)


I've tried to do it like above, but somehow it still doesn't work 

**Should I go with this approach??** 
![Zrzut ekranu 2019-08-6 o 07 58 26](https://user-images.githubusercontent.com/27632432/62514619-0e4e7d80-b820-11e9-8b74-08eef42c78b7.png)